Framework/PythonInterface/test/python/plugins/algorithms/SavePlot1DAsJsonTest.py

```diff
@@ -15,7 +15,7 @@ def test_save_one_curve(self):
         """ Test to Save one curve
         """
         datawsname = "constant energy cut"
-        E, I, err = self._createOneQCurve(datawsname)
+        E, I, err, dQ = self._createOneQCurve(datawsname)
```

Comment:
Shall we change the variable "E" in this test method to "Q" to make it more consistent?